### PR TITLE
added navigation to banned page, added turn options server side

### PR DESCRIPTION
**Pull request by devongillis**
after expand poker is merged i will link buttons to turn options

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -60,7 +60,9 @@ io.on('connection', (socket) => {
         if (result.matchFound && result.userData.isLoggedIn) {
             socket.emit("alert text", "You've already logged in. Please sign out of other sessions before trying again.");
         } else if (result.matchFound && result.banned) {
-            socket.emit("banned");
+            let updatedUser = UserUtils.updateUserLoginInfo(result.userData, true, socket.id);
```
Comment (rzayevan) [inline server/app.js:63]: What happens when the user wants to leave the banned page? Does their login status update to "false"?

Comment (devongillis) [reply server/app.js:63]: yes

Comment (devongillis) [reply server/app.js:63]: to clarify if they refresh or go back then it logs them out, but the navbar functionally is different throughout the website. on some pages its not responsive, and on others it navigates out but does/doesn't log people out.

Comment (rzayevan) [reply server/app.js:63]: That'll be fixed in my PR hopefully

**Review by rzayevan — APPROVED** [review]

**Review by darbrius — APPROVED** [review]

**Review by wpasadilla — APPROVED** [review]

**Pull request merged**